python/ray/data/_internal/bulk_dataset_iterator.py

```diff
@@ -50,6 +51,9 @@ def iter_torch_batches(
         dtypes: Optional[Union["torch.dtype", Dict[str, "torch.dtype"]]] = None,
         device: Optional[str] = None,
         drop_last: bool = False,
+        collate_fn: Optional[
+            Callable[[Union[np.ndarray, Dict[str, np.ndarray]]], "TorchTensorBatchType"]
```

Comment:
Shouldn't this now be like this?
```suggestion
            Callable[[Union[np.ndarray, Dict[str, np.ndarray]]], Any]
```

Reply:
updated